plugins/poll/app/models/poll.rb

```diff
@@ -78,6 +78,7 @@ def can_see_voters?(user)
 #  created_at       :datetime         not null
 #  updated_at       :datetime         not null
 #  chart_type       :integer          default("bar"), not null
+#  groups           :string
```

Comment:
I wonder if we should instead use an array of group_ids rather than a list of group names... 🤔

Reply:
We had a discussion about this and we realized that there will be some conversions anyway, from group names to group IDs and `.to_i`s or `parseInt`s. I see no advantage or disadvantage to either of the options.